applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -301,6 +301,33 @@ static int random_static_addr_cfg(void)
 	return -ENXIO;
 }
 
+int bt_mgmt_pa_sync_delete(struct bt_le_per_adv_sync *pa_sync)
+{
+	int ret;
+
+	if (IS_ENABLED(CONFIG_BT_PER_ADV_SYNC)) {
```

Comment:
Could do
```suggestion
	if (IS_ENABLED(CONFIG_BT_PER_ADV_SYNC)) {
		int ret;
```

Similarly for `bt_mgmt_conn_disconnect` below